dept44-models/src/test/java/se/sundsvall/dept44/models/api/paging/validation/impl/ValidSortByPropertyConstraintValidatorTest.java

```diff
@@ -55,19 +53,21 @@ void isNotValid() {
 		final var parameters = new TestParameters();
 		parameters.setSortBy(List.of("notASortableField"));
 		assertThat(validator.validate(parameters))
-			.first()
-			.extracting(ConstraintViolation::getMessage)
-			.isEqualTo("One or more of the sortBy properties [notASortableField] are not valid. Valid properties to sort by are [myField, id, mySecondField].");
+				.first()
+				.extracting(ConstraintViolation::getMessage)
+				.isEqualTo(
+						"One or more of the sortBy properties [notASortableField] are not valid. Valid properties to sort by are [myField, id, mySecondField].");
```

Comment:
Again.... the previous formating-rule must be the preferred?

`.isEqualTo("One or more of the sortBy properties [notASortableField] are not valid. Valid properties to sort by are [myField, id, mySecondField].");`

vs.
```
.isEqualTo(
						"One or more of the sortBy properties [notASortableField] are not valid. Valid properties to sort by are [myField, id, mySecondField].");
```